### ISSUE TITLE
Reckoning bombs are being removed after death

### ISSUE THREAD

**Issue by Mostprovidedlinks** [elysium-project/server#1318]
Issue: Reckoning bombs are being removed when dying

Fix: Reckoning bombs should remain after ressurecting

https://www.youtube.com/watch?v=z_Yftwjbl5U

lterally everything written by lothaermas said it didnt get removed even from dueling etc, it should remain

**Comment by stesve** [comment]
What info does the video provide? I can't find the connection? 

http://blue.cardplace.com/cache/wow-paladin/1069149.htm

```
The following things can cause you to drop your charges: 

 * Ending a Duel 
 * Mounting up (including gryphons) 
 * Autoattacking too early (e.g. from out of range) 
 * Switching Continents 
 * Switching Weapons 
 * Losing control of your character (Fear, Scattershot, Blind, Polymorph, Mind Control, etc.) 
```

I'm quite sure you could add death to that list.

**Comment by stesve** [comment]
This is from the reckcounter addon:

>      if ( event == "PLAYER_DEAD" ) then
>         auto_attack = false;
>         reckcounter_reset("Player has died.");
>     end

I'll assume that the person that wrote the addon, had done the proper investigation and gotten feedback if it was wrong.

**Comment by StadenElysium** [comment]
`You will however retain reckoning charges after casting Blessing of Protection on yourself or Divine Shield.`

I don't see why you'd be able to keep them after dying. Closing this issue if there aren't any sources.

**Comment by theloras** [comment]
@stesve - that blue post has been proven wrong on multiple fronts fyi

**Comment by stesve** [comment]
Come on Theloras.

> •go to the 2:17 mark and you will see that Mobywan kept his reck charges even while mounted which he then drops on the druid

Nope, he doesn't drop any charges.

**Comment by theloras** [comment]
not sure if you are saying that you agree with the video or not @stesve with your comment

**Comment by stesve** [comment]
> not sure if you are saying that you agree with the video or not @stesve with your comment
I'm saying; he doesn't drop any charges on the druid.

**Comment by theloras** [comment]
so you agree that the blue post is suspect at best given the video evidence?

**Comment by stesve** [comment]
The video shows no charges.

**Comment by theloras** [comment]
umm, wtf you smokin - he hits the druid with a full reck bomb at 2:32 - 
https://cdn.discordapp.com/attachments/253446111753273354/332664930828550144/wtf_are_you_blind_reck_bomb.jpg
that clip sequence begins with him **_MOUNTED_**

**Comment by stesve** [comment]
> umm, wtf you smokin - he hits the druid with a full reck bomb at 2:32 -
https://cdn.discordapp.com/attachments/253446111753273354/332664930828550144/wtf_are_you_blind_reck_bomb.jpg
 that clip sequence begins with him MOUNTED

We can agree on that this is a TBC video?

**Comment by theloras** [comment]
dude, do you really mean to imply that reckoning remained a PvP talent in TBC?

is that your theory?

**Comment by stesve** [comment]
> dude, do you really mean to imply that reckoning remained a PvP talent in TBC and wasn't changed into a tanking talent?

is that your theory?

I'm just saying thats a TBC video and reckoning didn't have charges since 2.0.1. 

You implying that im smokin "something" and blind seems to be same pattern you follow with every new crazy theory you dig up without proper research.

**Comment by theloras** [comment]
cuz he had the clip saved from vanilla maybe?!?!?

**Comment by theloras** [comment]
but be my guest and refute this video which you din't even bother commenting on...

Can I sit down to force a crit and gain a charge?
Charges will not generate off auto attacks or npc attacks by trying to sit down and force a crit. However, ability crits from physical abilities such as Sinister Strike, Hamstring, Auto-shot, Aimed shot, etc will generate a charge if you're sitting.

How to be a Gankadin
Published on Aug 8, 2006
https://www.youtube.com/watch?v=kuPEzEoexLI

**Comment by theloras** [comment]
TLDR - you ain't gonna win a Paladin Theorycrafting fight with me boy

**Comment by stesve** [comment]
> cuz he had the clip saved from vanilla maybe?!?!?

jesus, it's like pulling teeth with you...

Arena in vanilla? Thats new....

**Comment by theloras** [comment]
for christ sake man - he's got footage from both TBC and Vanilla - why is that such a difficult concept for you to get through your mind?

but still waiting for your amazing theorycrafting on this

How to be a Gankadin
Published on Aug 8, 2006
https://www.youtube.com/watch?v=kuPEzEoexLI

**Comment by stesve** [comment]
> for christ sake man - he's got footage from BOTH TBC and Vanilla - why is that such a difficult concept for you to get through your mind?

But you choose the part from TBC. Apparently with little research or insight.

> but still waiting for your amazing theorycrafting on this
I will watch it in a sec.

**Comment by theloras** [comment]
ITS FROM VANILLA YOU RETARD - HE'S LEVEL 60 IN THAT CLIP - WTF IS WRONG WITH YOU?!?!?

**Comment by stesve** [comment]
> ITS FROM VANILLA YOU RETARD - HE'S LEVEL 60 IN THAT CLIP - WTF IS WRONG WITH YOU?!?!?

Seriously. You are the guy the want stackable judgements and apparently now reckoning stacks to remain through death. I'm clearly not the retard here.

But i was told i would get this kind of response from you. .

I'm stopping here. This will never happen.... Such a waste of time.

**Comment by theloras** [comment]
you didnt even know that the talent was changed in TBC you dumb ass - and you can't even watch a video showing reck charges remaining after being mounted you dumb kunt

**Comment by stesve** [comment]
Reckoning charges doesn't remain through death. :)

EOD.

**Comment by stesve** [comment]
You should stop with the namecalling. I've said it before. It doesn't help your crusade.

**Comment by theloras** [comment]
whether reck charges remain after death or not is a moot point - that's for @StadenElysium to decide

im refuting your point that that blue post is the end all be all - it's been categorically shown to be false - i have 2 vanilla era videos showing that and you cannot deny the evidence

**Comment by stesve** [comment]
Nah, reck charges doesn't remain after death. There is nothing to decide.

EO stackable LOL... Take care.

**Comment by theloras** [comment]
good thing you aren't in any position of authority here!

**Comment by stesve** [comment]
> you didnt even know that the talent was changed in TBC you dumb ass - and you can't even watch a video showing reck charges remaining after being mounted

I wrote how it was changed in tbc 2 mins before you edited your post.
Seems like you didnt know how it worked?

**Comment by theloras** [comment]
don't even start WITH ME - you're the dumb ass who didn't know - watch the video again and read what mobywan said in the wall of text at the end...

ENJOY THE OLDSKOOL CLIP(S)!

**Comment by theloras** [comment]
Still waiting for your amazing comback to refute this btw

_Can I sit down to force a crit and gain a charge?
Charges will not generate off auto attacks or npc attacks by trying to sit down and force a crit. However, ability crits from physical abilities such as Sinister Strike, Hamstring, Auto-shot, Aimed shot, etc will generate a charge if you're sitting._

**How to be a Gankadin
Published on Aug 8, 2006**
https://www.youtube.com/watch?v=kuPEzEoexLI

**Comment by theloras** [comment]
So just to confirm what can be clearly seen in Mobywan's video:

- he types "ENJOY THE OLDSKOOL CLIP(S)!"
- he's using Grand Marshal Claymore
- he has Tier 2.5 Avenger Shoulders
- it even says LEVEL 60 in the character window
- he drops a Reck bomb to kill the druid

Obviously this clip is showing TBC footage...

**Comment by Citano** [comment]
@theloras it could have been from patch 2.0.1 footage that lasted about a month before TBC launched, however from the looks of this video it seems rather unlikely. Mobywan was rank 14 on retail vanilla.
https://worldofwarcraft.com/en-gb/character/frostmane/mobywan/achievements/legacy/player-vs-player

Also at 4:20 in Mobywan's video, he is mounted and he runs in and reckbombs the mage. However we do not see him mount up and dismount while keeping stacks. We only see him already mounted and possibly get stacks while mounted.

Still need source for stacks regarding death, both in terms of if death causing you to lose stacks and if death causing you to keep stacks. Seems to be tough to find source for either aside from the addon coding posted earlier.
Regarding death and loss of character, you do lose control of your character upon death until you release spirit or get resurrected. Stun should be counted as loss of control and it's fairly known that reckoning stacks remain (and can increase) while stunned.

**Comment by theloras** [comment]
@Citano - vanilla Reck Bombs weren't possible on the pre TBC 2.0.1 patch test server nor the live prepatch either since the talent at that point had been changed to the 5/5 flat 10% chance to proc rather than the 5/5 100% chance on crit chance.

Likewise, every Paladin with a juicy 2hander went 41+ in the Ret tree at that point anyway to grab Crusader Strike :)

This leaves us with the only alternative that it was a leftover clip from Mobywan's actual retail Vanilla playing as it shows Reckoning in its original form and not from TBC or 2.0.1 prepatch either.

For @stesve to throw around that blue post link in everyone's face and then claiming that the clips from Mobywan's video are not from Vanilla is just beyond the pale. Notice that he didn't bother replying to my multiple requests for him to find fault in the "How to be a Reckadin video" - which clearly show the /sit mechanic working during retail Vanilla?

Similarly, the ReckCounter addon still maintains Reck charges after death FYI which is the case here on Elysium and was the same on Nost.

**Comment by stesve** [comment]
If you can keep on topic and behave just slightly less childish, we can keep this discussion going.

> Similarly, the ReckCounter addon still maintains Reck charges after death FYI which is the case here on Elysium and was the same on Nost.

This seems very untrue. I've downloaded 4 versions of the addon. Everyone has the on death event - even with clear comments stating that player death reset charges.

**Comment by theloras** [comment]
dude, you were not even able to see Mobywan 1shot the druid with a Reck bomb in his video, nor did you even realize that that clip showed footage from vanilla not TBC, nor did you realize that the Reckoning talent was completely overhauled from 2.0.1 onwards to a tanking talent and you're calling me childish?

but i digress...

the charges remain lit up regardless of being alive or not

this has been the case for years now - from feenix to nost to vanilla gaming to elysium

and on feenix you maintained your charges after death

**Comment by stesve** [comment]
> the charges remain lit up regardless of being alive or not
> 
> this has been the case for years now - from feenix to nost to vanilla gaming to elysium
> 
> and on feenix you maintained your charges after death

Can you link one copy of the addon and the source from where its downloaded, that doesnt reset on death?

**Comment by theloras** [comment]
it's not hard to find - it's been in my guide since day one:

http://www.classicwow.com/forum/topic/66225-theloras-retribution-guide/

https://forum.nostalrius.org/viewtopic.php?f=43&t=34979

https://forum.elysium-project.org/topic/23030-theloras-retribution-guide/

http://www.mediafire.com/download/3aw939ny2a1f2i2/ReckCounter.rar

**Comment by stesve** [comment]
> it's not hard to find - it's been in my guide since day one:
> 
> http://www.classicwow.com/forum/topic/66225-theloras-retribution-guide/
> 
> https://forum.nostalrius.org/viewtopic.php?f=43&t=34979
> 
> https://forum.elysium-project.org/topic/23030-theloras-retribution-guide/
> 
> http://www.mediafire.com/download/3aw939ny2a1f2i2/ReckCounter.rar

Without analyzing the code to much. I can say that your addon states to be version 1.3a, but the file is modified and is not the same as the original 1.3a released Nov 2, 2006.

https://wow.curseforge.com/projects/project-1032

If you are using addons as a source for vanilla mechanics, make sure you use a version thats made for vanilla wow and not one that could be modified for nonworking mechanics on a private server.

From the real 1.3a source:

```
    this:RegisterEvent("PLAYER_ENTER_COMBAT"); -- Events below are for resetting ReckCounter
    this:RegisterEvent("PLAYER_LEAVE_COMBAT");
    this:RegisterEvent("PLAYER_DEAD");
```

and


```
    if ( event == "PLAYER_DEAD" ) then
        auto_attack = false;
        reckcounter_reset("Player has died.");
    end
```

**Comment by theloras** [comment]
are you suggesting that a Paladin would lose Reck charges by leaving combat or changing targets?

  if ( event == "PLAYER_ENTER_COMBAT" ) then
        auto_attack = true;
    end   

    if ( event == "PLAYER_LEAVE_COMBAT" ) then
        auto_attack = false;
        reckcounter_reset("Player changed targets after activating auto attack.");
    end

    if ( event == "PLAYER_DEAD" ) then
        auto_attack = false;
        reckcounter_reset("Player has died.");
    end

cause that's also what it says in the addon - which I can confirm is complete BS

**Comment by stesve** [comment]
> are you suggesting that a Paladin would lose Reck charges by leaving combat or changing targets?

No i'm not. Feel free to look at the complete sourcecode. 

The enter_combat event is used to track autoattack. Do some research before you call out bs.

**Comment by theloras** [comment]
it even says in the addon this as well:

function update_reckcounter(reck)
    --Updates GUI to reflect reckonings stored
    --Shamelessly cut from ComboFrame.lua

the original addon was a knockoff of a rogue combo points addon which are in fact removed when a rogue switches targets

**Comment by stesve** [comment]
> it even says in the addon this as well:
> 
> function update_reckcounter(reck)
>  --Updates GUI to reflect reckonings stored
>  --Shamelessly cut from ComboFrame.lua
> 
> the original addon was a knockoff of a rogue combo points addon which are in fact removed when a rogue switches targets
> 
> do some research yourself
> 

Your only argument for having stacks remaining trough death was this addon. I don't have to do more research, you just discarded your only "evidence".

**Comment by theloras** [comment]
for the last time, whether reck charges remain after death or not is moot point - that is up for Staden to decide.

I myself do not know whether they remained or not but stated since my initial posts was that the ReckCounter addon circles remain coloured and active even after death - this was the case on feenix, nost, vanilla gaming and now elysium - whereas on feenix you maintained your stored charges.

What I am doing is refuting you and your attempts at throwing the blue post in everyone's face as the Word of God - it has been proven to be flawed on two separate accounts from the /sit mechanic or being able to maintain reck charges while mounted.

Speaking of which, you have still not replied to my multiple requests for a response of the How to be a Gankadin video...

Are you going to continue to deflect?

**Comment by Mostprovidedlinks** [comment]
First off Stesve you are wrong. A 2006 player has stated this.

Pala man
Hey lotharemus, did the death of your character remove the reckoning bomb charges?

And thank you for answering this Lotharemas﻿
Lotharemas
Lotharemas2 months ago
Not that i recall, you could even leave combat entirely and still have Reckoning stacked﻿

**Comment by Mostprovidedlinks** [comment]
I have even found proof showing scatter shot, and mounting and fear all do not remove reckoning bomb like your bluetracker states

**Comment by Mostprovidedlinks** [comment]
First off Stesve you are wrong. A 2006 reckadin player has stated this.
https://www.youtube.com/watch?v=z_Yftwjbl5U
Pala man
Hey lotharemus, did the death of your character or enterring a dueling remove the reckoning bomb charges?

And thank you for answering this Lotharemas﻿
Lotharemas
Lotharemas2 months ago
Not that i recall, you could even leave combat entirely and still have Reckoning stacked﻿

I have even found proof showing scatter shot, and mounting and fear all do not remove reckoning bomb like http://blue.cardplace.com states.

Therefore reckoning bomb should be unrestricted from all of the following due to the proof that 60% of them are nonsense until one is proven with evidence

I have proof showing

    Ending a Duel is wrong (have comment evidence)
    Mounting up (including gryphons) is wrong (have video evidence)
    Autoattacking too early (e.g. from out of range) (have video evidence)
    Losing control of your character (Fear, Scattershot, Blind, Polymorph, Mind Control, etc.)

fear blind and scattershot and any stuns/sap - have video evidence

I don't know what your silly bluecardplace or addon proof have to do with anything because they are all proven the opposite in videos.

I will be posting all these videos soon. Hopefully, reckoning will remain after death now.
